tests/test_split_libraries_lea_seq.py

```diff
@@ -150,10 +242,52 @@ def test_get_LEA_seq_consensus_seqs(self):
                                                       min_diff_in_clusters,
                                                       barcode_column,
                                                       reverse_primer_column)
-
         actual = function_call['Sample1']['AGCTACGAGCTATTGC']
         expected = 'AAAAAAAAAAAAAAAAAAA^AAAAAAAAAAAAAAAAAA'
         self.assertEqual(actual, expected)
+        # this call tests the second condition of if loop
+        # in the function get_consensus_seq_lookup
+        # i.e. select the majority sequence, as the cluster ratio
+        # between max_cluster/second_best_cluster in the fwd_read_data
+        # (and rev_read_data) is 3/1 > 2.5,
+        # so the function get_consensus will not be called
+
+        fn_call, _ = get_LEA_seq_consensus_seqs(self.get_cons_fwd_read_fh,
+                                                self.get_cons_rev_read_fh,
+                                                self.get_cons_mapping_fp,
+                                                self.temp_dir,
+                                                barcode_type,
+                                                barcode_len,
+                                                barcode_correction_fn,
+                                                max_barcode_errors,
+                                                min_consensus,
+                                                max_cluster_ratio,
+                                                min_difference_in_bcs,
+                                                fwd_length,
+                                                rev_length,
+                                                min_reads_per_random_bc,
+                                                min_diff_in_clusters,
+                                                barcode_column,
+                                                reverse_primer_column)
+
+        get_cons_actual = fn_call['Sample1']['AGCTACGAGCTATTGC']
+        get_cons_expected = 'AAAAAAAAAACAAAAAAAA^AAAAAAAAAATAAAAATA'
+        self.assertEqual(get_cons_actual, get_cons_expected)
+        # this call tests the third condition of if loop
+        # in the function get_consensus_seq_lookup
+        # i.e. calls the get_consensus function, as the cluster ratio
+        # between max_cluster/second_best_cluster in the get_cons_fwd_read_data
+        # (and get_cons_rev_read_data) is 2/1 ( < 2.5)
+        # so the majority sequence will not be selected
+
+        get_cons_actual = fn_call['Sample2']['AGCTACGCATCAAGGG']
+        get_cons_expected = 'AAAAAAAAAATAAAAAAAA^TTAAAAAAAAAAAAGAAAA'
+        self.assertEqual(get_cons_actual, get_cons_expected)
+
+        if len(fn_call) <= 1:
```

Comment:
This should be rewritten using `self.assertEqual` instead of raising an exception.